sat/util.py

```diff
@@ -661,12 +662,12 @@ def get_s3_resource():
             )
     except OSError as err:
         LOGGER.error(f'Unable to load configuration: {err}')
-        raise SystemExit(1)
+        raise S3ResourceCreationError(str(err))
     except ValueError as err:
         # A ValueError is raised when an invalid value is given to boto3.resource,
         # e.g. an endpoint URL that is not a valid URL.
         LOGGER.error(f'Unable to load S3 API: {err}')
-        raise SystemExit(1)
+        raise S3ResourceCreationError(str(err))
```

Comment:
Similar to the above comment:

```suggestion
        raise S3ResourceCreationError(f'Unable to load S3 API: {err}') from err
```